examples/examples/cacheable_response.rs

```diff
@@ -44,7 +44,7 @@ async fn main() -> Result<(), CacheError> {
         .with_env_filter(filter)
         .init();
 
-    let backend = RedisBackend::new().await.unwrap().start();
+    let backend = RedisBuilder::single_new().await.unwrap().finish().start();
```

Comment:
To me, `standalone Redis` sounds much better than `single Redis`. What do you think?

Reply:
Naming was my weak point. I ain't against this name and agree with you. One question - rename all of these inclusions?